servlet.py

```diff
@@ -212,14 +258,25 @@ def get(self):
                 if self.get_arguments('include_deprecated_codes'):
                     responseData.append({'sourceLanguage': toAlpha2Code(l1), 'targetLanguage': toAlpha2Code(l2)})
             self.sendResponse({'responseData': responseData, 'responseDetails': None, 'responseStatus': 200})
+        elif query == 'chains':
+            src = self.get_argument('src', default=None)
```

Comment:
Is "src" what the other endpoints use or is there a more standard term we've been using?